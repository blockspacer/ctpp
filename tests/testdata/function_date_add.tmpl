// Start

Date: <TMPL_var icu_format_date("dd-MM-yyyy", date_add("2017-04-11","1day"))>

Time: <TMPL_var icu_format_date("dd-MM-yyyy hh:mm:ss", date_add("2017-04-11 12:23:34","1day"))>

Date: <TMPL_var icu_format_date("dd-MM-yyyy", date_add("2017-04-11","1week"))>

Time: <TMPL_var icu_format_date("dd-MM-yyyy hh:mm:ss", date_add("2017-04-11 12:23:34","1week"))>

Date: <TMPL_var icu_format_date("dd-MM-yyyy", date_add("2017-04-11","1month"))>

Time: <TMPL_var icu_format_date("dd-MM-yyyy hh:mm:ss", date_add("2017-04-11 12:23:34","1month"))>

Date: <TMPL_var icu_format_date("dd-MM-yyyy", date_add("2017-04-11","1year"))>

Time: <TMPL_var icu_format_date("dd-MM-yyyy hh:mm:ss", date_add("2017-04-11 12:23:34","1year"))>

Date: <TMPL_var icu_format_date("dd-MM-yyyy", date_add("2017-04-11","20days"))>

Time: <TMPL_var icu_format_date("dd-MM-yyyy hh:mm:ss", date_add("2017-04-11 12:23:34","20days"))>

// End.
