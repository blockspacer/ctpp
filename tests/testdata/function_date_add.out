// Start

Date: 12-04-2017

Time: 12-04-2017 12:23:34

Date: 18-04-2017

Time: 18-04-2017 12:23:34

Date: 11-05-2017

Time: 11-05-2017 12:23:34

Date: 11-04-2018

Time: 11-04-2018 12:23:34

Date: 01-05-2017

Time: 01-05-2017 12:23:34

// End.
